cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -288,8 +288,21 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) caerrors.AutoscalerErr
 	a.DebuggingSnapshotter.StartDataCollection()
 	defer a.DebuggingSnapshotter.Flush()
 
-	unschedulablePodLister := a.UnschedulablePodLister()
-	scheduledPodLister := a.ScheduledPodLister()
+	scheduledAndUnscheduledPodLister := a.ScheduledAndUnscheduledPodLister()
```

Comment:
nit: let's insert this whole logic after call to obtain node list, as it was before

Reply:
Done